### DISCUSSION TITLE
1.18.2无法启动

### DISCUSSION THREAD

**Discussion by Griseo-bh3rd** [Meloong-Git/PCL#1229] [提问] [answered]
1.18.2无法启动，但是1.19可以，不知道时什么原因，有好心人能帮忙看下错误报告吗qwq
ps：~~我朋友没更新PCL所以还是2.4.4，我也不知道为啥他不更新~~
[错误报告-2023-2-9_20.30.55.zip](https://github.com/Hex-Dragon/PCL2/files/10708895/-2023-2-9_20.30.55.zip)

**Comment by LTCatt** [comment]
更新看看……

**Comment by Griseo-bh3rd** [comment]
[错误报告-2023-2-12_11.31.58.zip](https://github.com/Hex-Dragon/PCL2/files/10724045/-2023-2-12_11.31.58.zip)

**Reply by Griseo-bh3rd** [reply]
> 崩溃报告里面说找不到支持库……不过总感觉不太对劲？
> 
> ```
> Failed to find module info for F:\??\C:\Program Files (x86)\360\360Safe\safemon\SafeWrapper.dll
> ```
> 
> ~MC 支持库怎么搜到 360 氨醛卫士那里了~

我不到啊，可能是我朋友把冈易的Java用360移到F盘的原因？
回头让他装个Java17看看qwq...

**Comment by LTCatt** [comment, marked as answer]
我寻思看起来是 360 的锅，F:\??\C:\Program Files... 看起来是 360 移文件夹移炸了

**Comment by Griseo-bh3rd** [comment]
出现了奇葩的情况，用自己装的Java也不行qwq。。。
[错误报告-2023-2-22_18.04.22.zip](https://github.com/Hex-Dragon/PCL2/files/10825138/-2023-2-22_18.04.22.zip)
[错误报告-2023-2-22_18.05.04.zip](https://github.com/Hex-Dragon/PCL2/files/10825140/-2023-2-22_18.05.04.zip)

**Comment by 212112111zwl** [comment]
换一个好点的防病毒软件？~

**Comment by Griseo-bh3rd** [comment]
漂亮，换个mc安装文件夹解决问题

**Reply by Chiloven945** [reply]
换火绒，没什么太大需求defender就ok
